Assets/AnkrDemo/Scripts/WebGLConnectionController.cs

```diff
@@ -1,52 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AnkrDemo.Scripts;
```

Comment:
Fix namespace - .Scripts folder should not contribute to the namespace

Reply:
Isn't only my scripts use this namespace. I think it shouldn't be part of my task.

Reply:
It is not a hard task to press f2 and rename the namespace